src/System.Runtime.Extensions/tests/AssemblyResolveTestApp/Class1.cs

```diff
@@ -5,7 +5,8 @@
 using System;
 using System.Reflection;
 using System.Threading;
-namespace AssemblyResolveTests
+
+namespace AssemblyResolveTestApp
```

Comment:
The previous name conflicted with the arcade versioning schema: `AssemblyResolveTests`.